internal/validate/validate.go

```diff
@@ -109,9 +109,10 @@ func Credentials() error {
 		configCMD += " init"
 	}
 	return fmt.Errorf(
-		"%w\n\nTo set credentials, run: %s %s",
+		"%w\n\nTo login using your Atlas username and password, run: %s auth login\nTo set credentials using API keys, run: %s %s",
```

Comment:
"log in" when used as a verb vs "login" as a noun :) thanks!
```suggestion
		"%w\n\nTo log in using your Atlas username and password, run: %s auth login\nTo set credentials using API keys, run: %s %s",
```

Reply:
updated